compact.go

```diff
@@ -514,7 +516,7 @@ func (c *LeveledCompactor) populateBlock(blocks []BlockReader, meta *BlockMeta,
 			allSymbols[s] = struct{}{}
 		}
 
-		all, err := indexr.Postings(allPostingsKey.Name, allPostingsKey.Value)
+		all, err := indexr.Postings("", "")
```

Comment:
Maybe would be nice to add some comment that this means fetching postings for all labels?

Reply:
Yea, there was some kind of awkwardness/conflict during refactoring. In absence of a better idea I just reverted to this. Will think again on how to solve it.